sass/components/_info-card.scss

```diff
@@ -43,4 +28,25 @@
       width: 24rem;
     }
   }
+}
+
+@media only screen and (min-width: 992px) {
+  .info-card {
+    align-items: flex-start;
+    flex-direction: row;
+    margin-right: 5.6rem;
+    text-align: left;
+
+    &__icon {
+      @media only screen and (min-width: 992px) {
+        margin-right: 1.5rem;
+      }
+    }
+
+    &__description {
+      @media only screen and (min-width: 992px) {
```

Comment:
you do not need to repeat this media query in line 47

Reply:
Forgot to delete this two lines...